ui/dashboards/src/components/GridLayout/GridLayout.tsx

```diff
@@ -21,21 +21,26 @@ import { GridTitle } from './GridTitle';
 const ResponsiveGridLayout = WidthProvider(Responsive);
 
 export interface GridLayoutProps extends BoxProps {
+  index: number;
   definition: GridDefinition;
-  renderGridItemContent: (definition: GridItemDefinition) => React.ReactNode;
+  renderGridItemContent: (definition: GridItemDefinition, groupIndex: number) => React.ReactNode;
 }
 
 /**
  * Layout component that arranges children in a Grid based on the definition.
  */
 export function GridLayout(props: GridLayoutProps) {
   const {
+    index,
     definition: { spec },
     renderGridItemContent,
     ...others
   } = props;
 
-  const [isOpen, setIsOpen] = useState(spec.display?.collapse?.open ?? true);
+  const [isOpen, setIsOpen] = useState(!!spec.display?.collapse?.open);
```

Comment:
Nit: This might be clearer with just `??` for the default (e.g. `spec.display?.collapse?.open ?? false`). The `!!` syntax always makes my eyes cross a little. 😂

Reply:
oh haha i actually like `!!` syntax because it looks less redundant